vars/main.yml

```diff
@@ -121,7 +121,7 @@ postgresql_parameters:
   - { option: "max_files_per_process",               value: "4096"    }
   - { option: "archive_mode",                        value: "on"      }
   - { option: "archive_timeout",                     value: "1800s"   }
-  - { option: "archive_command",                     value: "cd ."    } # not doing anything yet with WAL-s
+  - { option: "archive_command",                     value: "cd ."    } # If you use wal-g, then use: wal-g wal-push '%p'
```

Comment:
no comment needed here.
leave prepared parameter for wal-g, but comment out by default, example:
```
  - { option: "archive_command",                     value: "cd ."    } # not doing anything yet with WAL-s
#  - { option: "archive_command",                     value: "wal-g wal-push '% p'" } # archive WAL-s using WAL-G
```